### PR TITLE
Emit configuration file for bindings generation

### LINKED ISSUE #2358: Emit bindings config file in compiler
After #2349  we can generate bindings to be used from another smart-contract. It would be convenient to create configuration file during compilation, similar to how we process debug info and manifest.

Expected output should include:
1. For local struct parameters corresponding package from `go.mod`
2. For imported types, package taken from contract imports.

### PR DESCRIPTION
**Pull request by fyrchik**
Close #2358.
Depends on #2349 .

### CODE REVIEW

**Review by AnnaShaleva — APPROVED** [review]

File: cli/contract_test.go [hunk 1]
```diff
@@ -91,6 +92,119 @@ func TestCalcHash(t *testing.T) {
...
+	cmd = append(cmd, "--config", cfgPath,
+		"--out", filepath.Join(tmpDir, "out.nef"),
+		"--manifest", manifestPath,
+		"--bindings", bindingsPath)
+	e.Run(t, cmd...)
+	e.checkEOF(t)
+	require.FileExists(t, bindingsPath)
+
+	//d, _ := ioutil.ReadFile(bindingsPath)
+	//fmt.Println(string(d))
```
Comment (roman-khimov) [inline cli/contract_test.go:164]: Not really needed.

File: cli/smartcontract/smart_contract.go [hunk 2]
```diff
@@ -166,6 +166,10 @@ func NewCommands() []cli.Command {
 						Name:  "no-permissions",
 						Usage: "do not check if invoked contracts are allowed in manifest",
 					},
+					cli.StringFlag{
+						Name:  "bindings",
+						Usage: "file for smart-contract bindings configuration",
```
Comment (roman-khimov) [inline cli/smartcontract/smart_contract.go:171]: output file? I think we need to be more specific on what's the inputs and what's the outputs.

File: pkg/compiler/debug.go [hunk 3]
```diff
@@ -94,9 +98,10 @@ type DebugRange struct {
 
 // DebugParam represents variables's name and type.
 type DebugParam struct {
-	Name   string                  `json:"name"`
-	Type   string                  `json:"type"`
-	TypeSC smartcontract.ParamType `json:"-"`
+	Name     string                  `json:"name"`
+	Type     string                  `json:"type"`
+	RealType binding.Override        `json:"-"`
+	TypeSC   smartcontract.ParamType `json:"-"`
```
Comment (roman-khimov) [inline pkg/compiler/debug.go:104]: Don't we have some duplication in this structure now?

Comment (fyrchik) [reply pkg/compiler/debug.go:104]: No:
- `Type` is a type from `smartcontract/`
- `TypeSC` is a type from `vm/stackitem/`
- `RealType` is a type from the `Go` code.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**